### PR TITLE
Small fix for call stack extraction in trace_mgr.js.

### PR DESCRIPTION
**Pull request by mrkmarron**
##### Checklist

- [ ] `make -j4 test` (UNIX), or `vcbuild test` (Windows) passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows [commit guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#commit-message-guidelines)

### CODE REVIEW

**Review by MSLaguana — APPROVED** [review]

File: lib/trace_mgr.js [hunk 1]
```diff
@@ -488,7 +488,7 @@ function generateFuzzyStack(eventKind) {
       return directIsAbsolute(frame);
     });
 
-  const lastframe = errstk[errstk.length - 1];
+  const lastframe = (errstk.length !== 0 ? errstk[errstk.length - 1] : "empty_stack");
```
Comment (kfarnung) [inline lib/trace_mgr.js]: Nit: Should we just use `> 0` since it's more explicit?

**Review by kfarnung — APPROVED** [review]
LGTM

**Comment by kfarnung** [comment]
CI: https://ci.nodejs.org/job/chakracore-test-pull-request/214/

**Comment by kfarnung** [comment]
@mrkmarron Can you update the commit title to conform to the node rules?

* include a subsystem (in this case "lib")
* be shorter than 50 characters
* use a lowercase letter to start and no period

`lib: fix call stack extraction in trace_mgr.js`

File: lib/trace_mgr.js (hunk 1 above)
Comment (kfarnung) [reply lib/trace_mgr.js]: The CI failed with:

```
19:08:04 not ok 6 - /usr/home/iojs/build/workspace/node-test-linter/lib/trace_mgr.js
19:08:04   ---
19:08:04   message: Line 491 exceeds the maximum line length of 80.
19:08:04   severity: error
19:08:04   data:
19:08:04     line: 491
19:08:04     column: 1
19:08:04     ruleId: max-len
19:08:04   messages:
19:08:04     - message: Strings must use singlequote.
19:08:04       severity: error
19:08:04       data:
19:08:04         line: 491
19:08:04         column: 72
19:08:04         ruleId: quotes
19:08:04   …
```

**Review by kfarnung — CHANGES_REQUESTED** [review]
LGTM other than CI errors.

**Review by kfarnung — APPROVED** [review]

**Comment by kfarnung** [comment]
New CI: https://ci.nodejs.org/job/chakracore-test-pull-request/252/

**Comment by kfarnung** [comment]
@MSLaguana looks like the Ubuntu CI hit a seg fault: https://ci.nodejs.org/job/chakracore-test-linux/370/nodes=ubuntu1604-64/console

**Comment by kfarnung** [comment]
Linux rerun: https://ci.nodejs.org/job/chakracore-test-linux/371/

**Comment by kfarnung** [comment]
@MSLaguana it is consistent about where at least: https://ci.nodejs.org/job/chakracore-test-linux/371/nodes=ubuntu1404-64/console

**Comment by kfarnung** [comment]
Landed in 5e5a96c1003816fb63a7cfafc671a4ee53583b80

**Pull request merged**